code/game/objects/items/charter.dm

```diff
@@ -14,7 +14,7 @@
 		user << "The crew has already settled into the shift. It probably wouldn't be good to rename the station right now."
 		return
 
-	var/new_name = input(user, "What do you want to name [station_name()]? Keep in mind particularly terrible names may attract the attention of your employers.")  as text|null
+	var/new_name = stripped_input(user, "What do you want to name [station_name()]? Keep in mind particularly terrible names may attract the attention of your employers.")  as text|null
```

Comment:
They didn't seem to do much more processing that I found useful, so someone else can add it if they want.